src/axolotl/cli/train.py

```diff
@@ -44,7 +44,9 @@ def do_train(cfg, cli_args) -> None:
     else:
         dataset_meta = load_datasets(cfg=cfg, cli_args=cli_args)
 
-    model, tokenizer = train(cfg=cfg, cli_args=cli_args, dataset_meta=dataset_meta)
+    output = train(cfg=cfg, cli_args=cli_args, dataset_meta=dataset_meta)
+    model, tokenizer = output
+    # model, tokenizer = train(cfg=cfg, cli_args=cli_args, dataset_meta=dataset_meta)
```

Comment:
is this just for clarity or debugging? can remove the commented out line if we want to keep in this state.

Reply:
Whoops, was using this to debug, will revert.